lib/adroller/version.rb

```diff
@@ -1,3 +1,3 @@
 module Adroller
-  VERSION = '1.2.2'
```

Comment:
It would be pretty simple to make this backwards compatible, if that is important.  You could just make the all of the methods follow this pattern:

```
def value
  @value || ENV['value']
end
```
Other than that, looks good to me.